config/samples/flows_v1beta1_flowcollector.yaml

```diff
@@ -8,23 +8,26 @@ spec:
   agent:
     type: EBPF
     ebpf:
-      imagePullPolicy: IfNotPresent
+      imagePullPolicy: Always
       sampling: 50
       cacheActiveTimeout: 5s
       cacheMaxFlows: 100000
-      interfaces: [ ]
-      excludeInterfaces: [ "lo" ]
-      logLevel: info
+      interfaces: [ "eth0", "veth5871bad5" ]
```

Comment:
are we still going to be able to attach to all interfaces? (ie no explicit listing)

Reply:
So that feature is in progress,  See https://github.com/redhat-et/bpfd/issues/322#issuecomment-1460539896 for the specific issue!